otsdaq-utilities/MacroMaker/MacroMakerSupervisor.cc

```diff
@@ -1175,15 +1175,15 @@ void MacroMakerSupervisor::createMacro(HttpXmlDocument& /*xmldoc*/,
 {
 	__SUP_COUT__ << "MacroMaker wants to create a macro!!!!!!!!!" << __E__;
 	std::string Name     = CgiDataUtilities::postData(cgi, "Name");
-	std::string Sequence = CgiDataUtilities::postData(cgi, "Sequence");
+	std::string FESequence = CgiDataUtilities::postData(cgi, "FEsequence");
```

Comment:
This change should be reverted, because it is for the MacroMaker app, not the FEMacroTest app